### ISSUE TITLE
Can't sync with lan server after rejoining network

### ISSUE THREAD

**Issue by Gymcap** [bitfireAT/icsx5#218]
I'm running an http server locally at home which exposes my generated ics file to lan. I can link to it in ICSx5 easily, but once leaving the local network (leaving my house) the sync (obviously) fails to resolve.

After coming back home and reconnecting to the network however, the sync continues to fail to reaolve until I modify the entry.

I believe the app see's my link as unreachable, and sets some sort of flag to not check that link again, and It would be very useful in my usecase to be able to turn this off for this entry. If it fails to connect, id like it to try again on its next interval.

**Comment by ArnyminerZ** [comment]
This should not be happening, sync should always run, and the error should always be displayed, I believe. I will take a look and communicate what I find

**Comment by ArnyminerZ** [comment]
It should be working. Does syncing manually also fail? I mean, pulling down the screen forces the synchronization to run. Is it also failing?

**Comment by Gymcap** [comment]
Yes, it fails to reconnect until I modify the entry, like removing a character from the link, saving, and putting it back before saving again.

I'd like to reiterate that the link I'm using is only accessible when I am on my home network, and will (rightfully) fail to resolve if I'm not connected to it at sync time. But no, niether pulling to refresh nor the "synchronize now" fixes it. It seems once it fails to resolve once, it refuses to reconnect again until the entry is modified as described above.
